tests/test_random.py

```diff
@@ -965,21 +965,21 @@ def test_no_miss_numbers(self, dtype):
         assert_array_equal(actual_x, desired_x)
 
     @pytest.mark.parametrize("conv", [lambda x: dpnp.array([]),
-                                      lambda x: dpnp.asarray(x).astype(dpnp.int8),
-                                      lambda x: dpnp.asarray(x).astype(dpnp.float32),
+                                      lambda x: dpnp.astype(asarray(x), dpnp.int8),
+                                      lambda x: dpnp.astype(asarray(x), dpnp.float32),
                                       # lambda x: dpnp.asarray(x).astype(dpnp.complex64),
-                                      lambda x: dpnp.asarray(x).astype(object),
+                                      # lambda x: dpnp.astype(asarray(x), object),
                                       lambda x: dpnp.asarray([[i, i] for i in x]),
                                       lambda x: dpnp.vstack([x, x]).T,
                                       lambda x: (dpnp.asarray([(i, i) for i in x], [
                                                               ("a", int), ("b", int)]).view(dpnp.recarray)),
                                       lambda x: dpnp.asarray([(i, i) for i in x],
                                                              [("a", object), ("b", dpnp.int32)])],
                              ids=['lambda x: dpnp.array([])',
-                                  'lambda x: dpnp.asarray(x).astype(dpnp.int8)',
-                                  'lambda x: dpnp.asarray(x).astype(dpnp.float32)',
+                                  'lambda x: dpnp.astype(asarray(x), dpnp.int8)',
+                                  'lambda x: dpnp.astype(asarray(x), dpnp.float32)',
                                   # 'lambda x: dpnp.asarray(x).astype(dpnp.complex64)',
-                                  'lambda x: dpnp.asarray(x).astype(object)',
+                                  # 'lambda x: dpnp.astype(asarray(x), object)',
```

Comment:
It's better to add the test cases into skip list for disabled tests instead of adding it to the comments block in the test src.

Reply:
Looks like usm array does not support 'object' type, that it why this line is commented...